commands/audit/scarunner.go

```diff
@@ -381,3 +391,31 @@ func logDeps(uniqueDeps any) (err error) {
 
 	return
 }
+
+// This method will change the working directory to the scan's working directory.
+func buildDependencyTree(scan *utils.ScaScanResult, params *AuditParams) (*DependencyTreeResult, error) {
+	if err := os.Chdir(scan.WorkingDirectory); err != nil {
+		return nil, errorutils.CheckError(err)
+	}
+	treeResult, techErr := GetTechDependencyTree(params.AuditBasicParams, scan.Technology)
+	if techErr != nil {
+		return nil, fmt.Errorf("failed while building '%s' dependency tree:\n%s", scan.Technology, techErr.Error())
+	}
+	if treeResult.FlatTree == nil || len(treeResult.FlatTree.Nodes) == 0 {
+		return nil, errorutils.CheckErrorf("no dependencies were found. Please try to build your project and re-run the audit command")
+	}
+	return &treeResult, nil
+}
+
+func createXrayGraphScanParams(auditParams *AuditParams) *services.XrayGraphScanParams {
+	return &services.XrayGraphScanParams{
+		RepoPath:               auditParams.commonGraphScanParams.repoPath,
+		Watches:                auditParams.commonGraphScanParams.watches,
+		ScanType:               auditParams.commonGraphScanParams.scanType,
+		ProjectKey:             auditParams.commonGraphScanParams.projectKey,
+		IncludeVulnerabilities: auditParams.commonGraphScanParams.includeVulnerabilities,
+		IncludeLicenses:        auditParams.commonGraphScanParams.includeLicenses,
+		XscVersion:             auditParams.commonGraphScanParams.xscVersion,
+		MultiScanId:            auditParams.commonGraphScanParams.multiScanId,
+	}
+}
```

Comment:
This method could move to the AuditParams struct, since all attributes are related to it